WooCommerce/Classes/ViewModels/Order Details/OrderDetailsResultsControllers.swift

```diff
@@ -147,6 +163,22 @@ private extension OrderDetailsResultsControllers {
         try? productResultsController.performFetch()
     }
 
+    private func configureProductVariationResultsController(onReload: @escaping () -> Void) {
+        productVariationResultsController.onDidChangeContent = {
+            onReload()
+        }
+
+        productVariationResultsController.onDidResetContent = { [weak self] in
+            guard let self = self else {
+                return
+            }
+            self.refetchAllResultsControllers()
+            onReload()
+        }
+
+        try? productVariationResultsController.performFetch()
```

Comment:
good idea to log the error in case of failure even though other functions don't do this, added in: https://github.com/woocommerce/woocommerce-ios/pull/3197/commits/7dbe1f3ebf33f3e571285748bc76007225b1618f